stable/nfs-server-provisioner/README.md

```diff
@@ -75,7 +75,7 @@ their default values.
 | `storageClass.name`            | The name to assign the created StorageClass                                                                     | `nfs`                                                    |
 | `storageClass.allowVolumeExpansion` | Allow base storage PCV to be dynamically resizeable (set to null to disable )                              | `true                                                    |
 | `storageClass.parameters`      | Parameters for StorageClass                                                                                     | `{}`                                                     |
-| `storageClass.mountOptions`    | Mount options for StorageClass                                                                                  | `[ "vers=4.1", "noatime" ]`                              |
+| `storageClass.mountOptions`    | Mount options for StorageClass                                                                                  | `[ "vers=3", "relatime" ]`                               |
```

Comment:
```suggestion
| `storageClass.mountOptions`    | Mount options for StorageClass                                                                                  | `["vers=3"]`                               |
```

Reply:
solved in https://github.com/helm/charts/pull/19260/commits/71de49c4ddab4058591049876d3f7a1819dbef4a